src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/FunctionId.cs

```diff
@@ -484,10 +484,14 @@ internal enum FunctionId
         Renamer_FindRenameLocationsAsync = 387,
         Renamer_ResolveConflictsAsync = 388,
 
-        ChangeSignature_Data = 389,
+        ChangeSignature_Data = 400,
```

Comment:
>400 [](start = 31, length = 3)

Shouldn't this be immutable once shipped? This is used for telemetry, so changing the values would render counters that span multiple version useless.

Reply:
OK, as long as we haven't shipped any of these.

---
In reply to: [418199711](https://github.com/dotnet/roslyn/pull/43636#discussion_r418199711) [](ancestors = 418199711)